auto_install/install.sh

```diff
@@ -84,15 +84,29 @@ distro_check() {
   if hash lsb_release 2>/dev/null; then
       PLAT=$(lsb_release -si)
       OSCN=$(lsb_release -sc) # We want this to be trusty xenial or jessie
-  
-      if [[ $PLAT == "Ubuntu" || $PLAT == "Raspbian" || $PLAT == "Debian" ]]; then
-          if [[ $OSCN != "trusty" && $OSCN != "xenial" && $OSCN != "jessie" ]]; then
-              maybeOS_Support
-          fi
+
+      case ${PLAT} in
+          Ubuntu|Raspbian|Debian|Devuan)
```

Comment:
@fernandog because there are different actions to be taken depending on distro! 

For example, Static IP address is not handled by PiVPN if you are using Ubuntu || Debian because it can be a cloud instance with its own address. 

so makes total sense to have different conditions for different actions. 

I advise you to better read the code, so you can better understand what I am saying